app/models/user.rb

```diff
@@ -577,11 +577,30 @@ def banned_at_date?(date)
     delegate_roles.select { |role| role.metadata.status == RolesMetadataDelegateRegions.statuses[:senior_delegate] }
   end
 
-  private def groups_with_read_access
+  private def can_view_current_banned_competitors?
+    wdc_team? || ethics_committee? || board_member? || staff_delegate? || weat_team? || results_team? || admin?
+  end
+
+  private def can_view_past_banned_competitors?
+    wdc_team? || ethics_committee? || board_member? || weat_team? || results_team? || admin?
+  end
```

Comment:
In line with my other, longer comment about "access hierarchy" below: Can this be simplified to an implementation where `can_view_current` is simplified to `can_view_past || staff_delegate?`?

Reply:
Done